plugins/source/azure/resources/services/subscription/locations.go

```diff
@@ -14,16 +14,16 @@ func locations() *schema.Table {
 		Name:        "azure_subscription_subscription_locations",
 		Resolver:    fetchLocations,
 		Description: "https://learn.microsoft.com/en-us/rest/api/resources/subscriptions/list-locations?tabs=HTTP#location",
```

Comment:
Docs were already pointing to the right API responses so no change was made there.